release/rllib_tests/app_config.yaml

```diff
@@ -19,7 +19,10 @@ python:
   - ale-py==0.8.0
   - gym==0.26.2
   - mujoco-py<2.2,>=2.1
-  - autorom[accept-rom-license]
+  # AutoROM downloads ROMs via torrent when they are built. The torrent is unreliable,
+  # so we built it for py3 and use that instead. This wheel was tested for python 3.7, 3.8,
+  # and 3.9.
+  - https://ray-ci-deps-wheels.s3.us-west-2.amazonaws.com/AutoROM.accept_rom_license-0.5.4-py3-none-any.whl
```

Comment:
one thing, do we need to also install `autorom`? can't remember if this installs `autorom` or just the actual ROMs..

Reply:
This should be sufficient. Let's see if the release tests pass now on ci.